qemu/tests/hugepage_mem_stress.py

```diff
@@ -31,6 +35,16 @@ def heavyload_install():
 
     os_type = params["os_type"]
     verify_wait_timeout = params.get_numeric("verify_wait_timeout", 60)
+
+    if params.get("start_vm") == "no":
+        error_context.context("Check large memory pages free on host.",
+                              logging.info)
+        hp = test_setup.HugePageConfig(params)
+        if hp.get_target_hugepages() != utils_memory.get_num_huge_pages_free():
```

Comment:
This is not going to happen.
1. `target_hugepages` always equal to `HugePages_Free` if `nr_overcommit_hugepages = 0` or not hugepages leaked.
2. If hugepages leaked, test will fail.

Reply:
thanks a lot, modify after discussing with the feature owner.